src/burp/UuidIssue.java

```diff
@@ -19,37 +19,43 @@ public UuidIssue(IHttpRequestResponse baseRequestResponse,
 	}
 
 	@Override public String getIssueName() {
-		return "Version " + uuid.version() + " UUID in HTTP request";
+		int v = uuid.version();
+		if (v > 5 || v < 1)
+			return "Unknown version " + v + " UUID in HTTP Request";
+		else
+			return "Version " + v + " UUID in HTTP request";
 	}
 
 	@Override public String getIssueDetail() {
 		int v = uuid.version();
-		final String prefix = "The request contains the version " + v +
+		final String known_prefix = "The request contains the version " + v +
 			" UUID <b>" + uuid + "</b> which is ";
+        final String unknown_prefix = "The request contains an undefined version " + v +
+            " UUID <b>" + uuid + "</b> which is ";
```

Comment:
There's little benefit for this assignment since the only place it's used is the `default` branch (unlike `known_prefix`, which is used in the 5 others). I suggest construcing the whole string there in one step, which helps readability.